common/src/main/java/com/wynntils/functions/CombatFunctions.java

```diff
@@ -113,6 +113,18 @@ protected List<String> getAliases() {
         }
     }
 
+    public static class WasLastSpellShiftCastedFunction extends Function<Boolean> {
```

Comment:
```suggestion
    public static class IsLastSpellShiftCastedFunction extends Function<Boolean> {
```

Better to use present verbs here.

Reply:
Also, check how other functions define localization in en_us.json, and add it for this function as well.

Reply:
Oh I forgot about function definitions...
Done!